core/extensions/AppWithPlugin/ReadMe.md

```diff
@@ -17,3 +17,5 @@ By using `AssemblyDependencyResolver` along with a custom `AssemblyLoadContext`,
     - You should see the app output a list of installed commands.
 4. In the AppWithPlugin directory, use `dotnet run commandName` where `commandName` is one of the names that was outputted in the previous step.
```

Comment:
Shouldn't step 4 be deleted, so that step 5 replaces step 4. @jkoritzinsky?
You could also change `commandName1 commandName2` to `commandName1 [commandName2...]`